Automattic/wp-calypso
# Revert "FSE/Gutenberg: Add link color to block editor"

Reverts Automattic/wp-calypso#41992

#### Testing instructions (taken from #41992)

1. Make sure your sandbox is up-to-date and that it's using a Gutenberg plugin >=8.1.0.
1. Apply patch (D43689-code)
1. Sandbox `example.wordpress.com`
1. Apply theme "Independent Publisher 2", "Penscratch 2", or "TwentyTwenty"
1. Load the editor `/wp-admin/post-new.php`
1. Add a link in a paragraph, text should be blue (not black).

#### Screenshot

With TwentyTwenty theme

![after](https://user-images.githubusercontent.com/841763/81419211-302db580-914e-11ea-9834-fe4e03423bcf.png)
